config/locales/links/en.yml

```diff
@@ -2,6 +2,7 @@
 en:
   links:
     back_to_sp: "‹ Back to %{sp}"
+    back_to_sp_alt: "Back to %{sp}"
```

Comment:
What if we changed the other key to not have the arrow, and moved the arrow into the template? Or too much noise?